msteams-platform/bots/how-to/authentication/bot-sso-overview.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Overview to authentication using SSO in Teams with Azure AD
+description: Learn about Single sign-on (SSO) authentication in Teams and how to enable it in bots and message extension.
+ms.topic: conceptual
+ms.localizationpriority: high
+---
+# Enable SSO for your app
+
+With SSO in Teams, app users have the advantage of using Teams to access bot or message extension apps. After logging into Teams using Microsoft or Microsoft 365 account, app users can use your app without needing to sign in again. Your app is available to app users on any device with access granted through Azure AD.
+
+This section covers SSO configuration required for bot and messaging extension apps.
+
+## SSO user experience in Teams
+
+App users sign in to Teams using either personal Microsoft account or Microsoft 365 account. You can take advantage of this, and use SSO to authenticate and authorize the app users. Azure AD provides app users access to your bot or message extension app.
+
+- Teams authenticates and stores the identity of its app user.
+- Your bot uses the stored identity of the app user who is already validated by Teams.
+- The app user needs to give consent to Teams for using the identity to access for using your bot.
+- The app user can access the app on web, desktop, or mobile client.
+
+> [!NOTE]
+> App users can't give permission to some permission scopes, such as `Sites.ReadWrite.All`, which allows the app user to read and write to all SharePoint and OneDrive assets in the tenant. For such scopes, only the tenant administrator than grant consent on an app user's behalf.
+
+Now, let's see what happens at the backend during runtime to achieve SSO experience within Teams.
+
+## SSO in Teams at runtime
+
+Achieve SSO in a bot or message extension app by obtaining access token for the Teams app user who's currently logged in. This process involves the bot app client and server, Teams client, and Azure AD. During this interaction, the app user must give consent for using Teams identity to obtain the access token in a multi-tenant environment.
+
+The following image shows how SSO works when a Teams app user attempts to access the bot or message extension app:
+
+:::image type="content" source="../../../assets/images/authentication/teams-sso-bots/sso-runtime-seqd-bots.png" alt-text="Bots at runtime" lightbox="../../../assets/images/authentication/teams-sso-bots/sso-runtime-seqd-bots.png":::
```

Comment:
please validate preview if images appear right.

![image](https://user-images.githubusercontent.com/5934356/202571713-6864a94f-24d4-4333-93e5-a51a62f12636.png)

Reply:
Hello @pradeepananth, Thank you for the comment. The images are rendering for me, and other who are reviewing the PR. Please let me know if any error is showing up instead of the image.